atd-vzd/migrations/default/1715094801442_add_import_log_table/up.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE cris_import_log (
+    id SERIAL PRIMARY KEY,
+    object_path TEXT NOT NULL,
+    object_name TEXT NOT NULL,
+    created_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    completed_at TIMESTAMPTZ DEFAULT NULL,
+    records_processed JSONB DEFAULT '{}',
+    outcome_status TEXT NULL
```

Comment:
This column is intended to hold a short, textual message that indicates how the program finished. It will default to something like "nominal completion," and for all error states, each one can have a text string that goes into here that's a breadcrumb for the dev who is trying to figure out what went wrong and why. The breadcrumb strings can come from the string argument of the `raise()` function if we stick in some sort of error handling. I'm not 100% i know exactly how this goes down in python, but I'm pretty hopeful that it's possible.